depthai_demo.py

```diff
@@ -485,7 +491,7 @@ def create_nn(self):
                 nn.passthrough.link(self.nodes.xout_nn_input.input)
 
             if conf.args.sync:
-                if self.nn_manager.source == "color" and hasattr(self.nodes, "xout_rgb"):
+                if self.nn_manager.source in ["color", "host"] and hasattr(self.nodes, "xout_rgb"):
```

Comment:
I wouldn't like to mix xout_rgb to send the host frames. Let's add another if condition and check for `xout_host` (which we create prior)